client/components/tinymce/plugins/media/plugin.jsx

```diff
@@ -415,6 +415,28 @@ function mediaButton( editor ) {
 		}
 	} );
 
+	editor.addButton( 'wp_img_edit', {
+		tooltip: 'Edit',
+		icon: 'dashicon dashicons-edit',
+		onclick: function() {
+			const siteId = 125351479;
+			const node = editor.selection.getNode();
+			const m = node.className.match( /wp-image-(\d+)/ );
```

Comment:
Also does tinymce always return an element for `editor.selection.getNode()`? Or can that sometimes be null